user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
+                            break
+                if break_loop == 0:
+                    pass
+                else:
+                    break
+            else:
+                print("Not an acceptable answer for decision! Ending function")
+                return
+    conn.commit()
+    print(f"Finished create_collection function!!")
+
+
+
+def modify_collection(conn, user_ID) -> None:
+    """
+    modify a current collection that is in the incollection table 
+
+    :param conn: The connection to the database
+    :param user_ID: The ID of the user
+    :param movie_ID: The ID of the current movie
+    :return: Nothing
+    """
+    # A Query to get the collection ID for a certain User ID
+    query = """
+    SELECT collectionid FROM moviecollection AS mc
+    WHERE mc.madeby = %s
+    ORDER BY collectionid ASC
+    """
+    # A query to remove a movie from a collection
+    remove_movie_query = """
+    DELETE FROM incollection 
+    WHERE movieid = %s AND collectionid = %s
+    """
+    # A query to add a movie into a collection
+    add_movie_query = """
+    INSERT INTO incollection (movieid, collectionid)
+    VALUES (%s, %s)
+    """
+    # Deletes a collection from the table
+    delete_moviecollection_query = """
+    DELETE FROM moviecollection 
+    WHERE collectionid = %s
+    """
+    # Deletes all movie IDs associated with the collection ID
+    delete_all_movie_query = """
+    DELETE FROM incollection
+    WHERE collectionid = %s
+    """
+
+    # Changes the name of the collection
+    change_name_query = """
+    UPDATE moviecollection
+    SET name = %s
+    WHERE collectionid = %s
+    """
+
+    
+    collection_ID = ""
+    movie_ID = movie_funcs.browse_movies(conn)
+    if movie_ID == -1:
+        print(f"Function is end, couldn't find movie")
+        return
+    with conn.cursor() as curs:
+        action = ""
+        curs.execute(query, (user_ID,))
+        collection_ID_list = curs.fetchall()
+        # User Select which collection they want to modify based off of Index
+        print(f"Here are all of the collections for the user\n{collection_ID_list}")
+        index = int(input("Please input a number that is within the index of the list of collection IDs for the collection you want: "))
+        if index >= len(collection_ID_list) or index < 0:
+            print(f"Invalid index, ending function...")
+            return
+
+        # Start of Actions
+        print("what action would you like to do?")
+        action = int(input_utils.get_input_matching("1 - remove a movie\n2 - add a movie \n3 - modify name of collection\n4 - delete collection\n5 - exit function: "))
+        if action == 1 or action == 2 or action == 3 or action == 4 or action == 5:
+             match action:
+                # action for removing a movie
+                # This works
+                case 1:
+                    curs.execute(remove_movie_query, (movie_ID, collection_ID_list[index][0]))
```

Comment:
When selecting a movie to remove from the collection, you probably want to display a list of movies in the collection to browse rather than have the users browse all movies.